packages/lodestar/src/util/function.ts

```diff
@@ -1,15 +0,0 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
This function had a bunch of Typescript errors, but since it's not used anywhere I delete it.